inc/render/class-woo-comparison-block.php

```diff
@@ -0,0 +1,143 @@
+<?php
+/**
+ * Woo_Comparison_Block
+ *
+ * @package ThemeIsle\GutenbergBlocks\Render
+ */
+
+namespace ThemeIsle\GutenbergBlocks\Render;
+
+use ThemeIsle\GutenbergBlocks\Base_Block;
+
+/**
+ * Class Woo_Comparison_Block
+ */
+class Woo_Comparison_Block extends Base_Block {
+
+	/**
+	 * Woo_Comparison_Block constructor.
+	 *
+	 * @since   1.7.0
+	 * @access  public
+	 */
+	public function __construct() {
+		parent::__construct();
+		if ( ! 'valid' === apply_filters( 'product_neve_license_status', false ) || true !== apply_filters( 'neve_has_comparison_table_support', false ) || ! class_exists( 'WooCommerce' ) || ! class_exists( 'Neve_Pro\Modules\Woocommerce_Booster\Comparison_Table\View\Table' ) ) {
+			return;
+		}
+
+		if ( is_admin() && class_exists( '\Neve_Pro\Modules\Woocommerce_Booster\Comparison_Table\Main' ) ) {
+			$style_path = neve_is_new_skin() ? 'css/woocommerce' : 'css/woocommerce-legacy';
```

Comment:
I think, we don't need any new_skin check on here since we don't have comparison table support for old skin.